frontend/packages/network-attachment-definition-plugin/src/components/network-attachment-definitions/NetworkAttachmentDefinitionsForm.tsx

```diff
@@ -26,6 +32,21 @@ import {
 } from '../../types';
 import NetworkTypeOptions from './NetworkTypeOptions';
 
+const getModelAvailableVersion = (model: K8sKind): string =>
+  _.uniq([model.apiVersion, 'v1', 'v1alpha3', 'v1beta1', 'v1alpha1']).find(
+    (v) => !!modelFor(referenceForGroupVersionKind(model.apiGroup)(v)(model.kind)),
+  );
+
+export const networkReferenceForModel = (model: K8sKind): string => {
+  const version = getModelAvailableVersion(model);
+
+  if (version) {
+    return referenceForGroupVersionKind(model.apiGroup)(version)(model.kind);
+  }
+
+  return model.kind;
+};
+
```

Comment:
```suggestion
const hasHyperConvergeCRD = !!['v1beta1', 'v1alpha1']).find(
    (v) => !!modelFor(referenceForGroupVersionKind(HyperConvergedModel.apiGroup)(v)(HyperConvergedModel.kind)),
)
```
I think we only support 'v1beta1' and 'v1alpha1' we should not allow other apiVersions